wix-bookings/guides/About Wix Bookings.md

```diff
@@ -2,27 +2,44 @@
 
 [Wix Bookings](https://support.wix.com/en/article/about-wix-bookings) allows a business owner to set up a system to accept and manage bookings for services.
 
-Common Wix Bookings use cases:
+Some examples of Wix Bookings use cases are:
+
 - Beauty & wellness services: e.g., book a haircut or a massage
 - Fitness services: e.g., book a yoga class or a private training session
 - Education services: e.g., book a programming course or a personal math tutor
 
-The Wix Bookings APIs include:
+Learn more about about how site collaborators can [create and manage their bookings]. (https://support.wix.com/en/article/wix-bookings-about-wix-bookings)
+
+With the Wix Bookings APIs, you can:
+
 - **Services**: Manage the specific offerings this business provides
 - **Catalog**: View a list of all the different services this business provides
 - **Resources**: Manage each of the business staff members, locations/rooms, and equipment that can be “booked” or linked to a booking
 - **Bookings**: Manage bookings to individual services - create, update and cancel bookings for services
 - **Calendar**: A bird’s-eye view of the business calendar, where you can easily view sessions available for booking, sessions taught per staff member, etc.
 - **Schedules and Sessions**: Manage the individual schedules of each staff member, location, etc. (i.e., resource) and session 
 
-> **Important**:  
+# Before you begin
+
```

Comment:
I think this section should be a bulleted list, and the note should be part of the list